test/Sema/inaccurate-warnings-in-single-case-switch.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
This is a [SIL diagnostic](https://github.com/apple/swift/blob/8ab43a4a444702eb96e60ef78e4512947b16dbbf/lib/SILGen/SILGenPattern.cpp#L1069). Running semantic analysis won’t catch a regression.

Reply:
Oh! Yeah, that needs to be moved to test/SILGen then!